packages/block-editor/src/components/grid-visualizer/style.scss

```diff
@@ -13,7 +13,9 @@
 }
 
 .block-editor-grid-visualizer__item {
-	border: $border-width dashed $gray-300;
+	background-color: color-mix(in srgb, currentColor 10%, transparent);
+	outline: 1px solid color-mix(in srgb, currentColor 20%, transparent);
+	border-radius: 2px;
```

Comment:
Probably should use `$border-width` for the 1px and `$radius-block-ui` for the 2px.